src/js/providers/html5.js

```diff
@@ -419,9 +420,13 @@ define([
                 return;
             }
             _videotag.removeAttribute('src');
-            if (!_isIE) {
+            if (!_isMSIE) {
                 _videotag.load();
             }
+            // IE may continue to play a video after changing source and loading a new media file.  JW7-1705
+            if(utils.isIETrident()) {
```

Comment:
@jw-kaurand - good initiative to create a test-page which shows this behavior in Edge. 
We should post it as a question here : https://connect.microsoft.com/IE
and link the question here (as opposed to linking to a jw story, which is not visible to open-source community)